### ISSUE TITLE
Permission denied on Fedora 29 Silverblue

### ISSUE THREAD

**Issue by Zlopez** [flathub/com.bluejeans.BlueJeans#1]
When trying to start this flatpak on Fedora 29 Silverblue, I'm getting this:
```
$> flatpak run com.bluejeans.BlueJeans 
/app/bin/bluejeans: line 2: /app/extra/bluejeans: Permission denied
/app/bin/bluejeans: line 2: exec: /app/extra/bluejeans: cannot execute: Permission denied
```

**Comment by Zlopez** [comment]
Here is the output with -v option
```
$> flatpak run -v com.bluejeans.BlueJeans 
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening user flatpak installation at path /home/zlopez/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/zlopez/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/zlopez/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Regenerating ld.so.cache /home/zlopez/.var/app/com.bluejeans.BlueJeans/.ld.so/e779552b9a7635510ec54d2f55d6ea348dc4850ffe7f451c931f8aa29acbfff9
F: Running: 'bwrap --args 17 ldconfig -X -C /run/ld-so-cache-dir/e779552b9a7635510ec54d2f55d6ea348dc4850ffe7f451c931f8aa29acbfff9'
F: Cleaning up unused container id 3188406814
F: Allocated instance id 1186072098
F: Allowing wayland access
F: Allowing x11 access
F: Allowing pulseaudio access
F: Pulseaudio user configuration file '/home/zlopez/.config/pulse/client.conf': Error opening file /home/zlopez/.config/pulse/client.conf: No such file or directory
F: Running 'bwrap --args 33 /usr/libexec/flatpak-dbus-proxy --args=35'
F: Running 'bwrap --args 31 bluejeans'
/app/bin/bluejeans: line 2: /app/extra/bluejeans: Permission denied
/app/bin/bluejeans: line 2: exec: /app/extra/bluejeans: cannot execute: Permission denied
```

**Comment by AsciiWolf** [comment]
That's weird. The `/app/extra/bluejeans` file should have executable bit set by apply_extra during installation. It worked without any issues on my machine. Thanks for reporting, I'll look into it.

**Comment by AsciiWolf** [comment]
Works fine on my regular Fedora 29 Workstation system. I will test it on Silverblue when I get home.

**Comment by AsciiWolf** [comment]
I reproduced the issue! It seems to be present only if the Flatpak is installed system-wide. Everything is ok when it is installed with `--user` switch. @TingPing Any idea?

**Comment by AsciiWolf** [comment]
Fixed in 468bf69.
